.simplecov

```diff
@@ -4,17 +4,11 @@
 SimpleCov.start 'rails' do
   add_filter '/bin/'
   add_filter '/config/'
-  add_filter '/jobs/application_job.rb'
-  add_filter '/schemas/'
-  add_filter '/lib/generators'
-  add_filter '/spec/'
-  add_filter '/vendor/'
-  add_filter '/public'
-  add_filter '/swagger'
   add_filter '/script'
-  add_filter '/log'
   add_filter '/db'
-  add_filter '/lib/tasks/sample_data/'
+
+  # We haven't managed to make simplecov recognise rake coverage accurately.
+  add_filter '/lib/tasks/'
```

Comment:
Even though you said to ignore it, undercover seems to be warning about the new test you added: https://github.com/openfoodfoundation/openfoodnetwork/actions/runs/16640227560/job/47089363544

```
1) node `block (_t, args)` type: block,
      loc: lib/tasks/simplecov.rake:5:22, coverage: 0.0%
 2) node `block` type: block,
      loc: lib/tasks/simplecov.rake:18:21, coverage: 0.0%
```

Do you know why?

Reply:
I wonder if the preceding slash should be removed? The documentation doesn't include one: https://www.rubydoc.info/gems/simplecov/SimpleCov%2FConfiguration:add_filter

```suggestion
  add_filter 'lib/tasks/'
```

Reply:
Hmm, but the readme does, so it's probably not the problem here.